OrganizationSearch/OrganizationsListContainer.test.js

```diff
@@ -15,17 +22,41 @@ const renderOrganizationsListContainer = (mutator) => (render(
 ));
 
 describe('OrganizationsListContainer component', () => {
-  it('should not fetch organizations when plugin is open by default ', async () => {
-    const mutator = {
+  let mutator;
+
+  beforeEach(() => {
+    mutator = {
       organizationsListOrgs: {
-        GET: jest.fn(),
+        GET: jest.fn(() => Promise.resolve({ organizations: [], totalRecords: 0 })),
       },
     };
+  });
 
+  it('should not fetch organizations when plugin is open by default', async () => {
     await act(async () => {
       renderOrganizationsListContainer(mutator);
     });
 
     expect(mutator.organizationsListOrgs.GET).not.toHaveBeenCalled();
   });
+
+  it('should not fetch organizations since filters were not applied', async () => {
+    await act(async () => {
```

Comment:
Should the render be inside the `act()`?

Reply:
Otherwise, it returns ` Warning: An update to OrganizationsListContainer inside a test was not wrapped in act(...).`